KDL_DKIN:
  ros__parameters:
    joint_elbow:
    - 2.0
    - 0.0
    - 0.0
    - 0.0
    - 0.0
    - 0.0
    - 0.0
    - 0.0
    - 1.0
    joint_fixed: null
    joint_fixed_end:
    - -3.0
    - 0.0
    - 0.0
    - 0.0
    - 0.0
    - 0.0
    - 0.0
    - 0.0
    - 0.0
    joint_shoulder:
    - 0.0
    - 0.0
    - 5.0
    - 0.0
    - 0.0
    - 0.0
    - 0.0
    - 0.0
    - 1.0
    joint_wrist:
    - 3.0
    - 0.0
    - 0.0
    - 1.578
    - 0.0
    - 0.0
    - 0.0
    - 0.0
    - 0.0
    joint_wrist_connector:
    - 0.0
    - 0.0
    - 0.0
    - 0.0
    - 1.578
    - 0.0
    - 0.0
    - 0.0
    - 0.0
